//@+leo-ver=5-thin
//@+node:gcross.20110902105950.2685: * @file nutcracker.proto
//@@language cplusplus

package Nutcracker.Protobuf;

enum Normalization {
    MIDDLE = 0;
    LEFT = 1;
    RIGHT = 2;
}

message StateSite {
    optional Normalization normalization = 1;
    required uint32 physical_dimension = 2;
    required uint32 left_dimension = 3;
    required uint32 right_dimension = 4;
    repeated double data = 5 [packed=true];
}


message State {
    repeated StateSite sites = 1;
}
//@-leo
